test/integration/parachain/staging/setup/initialize.test.ts

```diff
@@ -187,4 +279,138 @@ describe("Initialize parachain state", () => {
         }
         assert.isTrue(isNominationEnabled);
     });
+
+    it("should set collateral ceiling and thresholds for aUSD", async function () {
+        // only get aUSD if it hasn't been set yet (e.g. when running this test in isolation)
+        !aUSD ? (aUSD = await getAUSDForeignAsset(sudoInterBtcAPI.assetRegistry)) : undefined;
+
+        if (aUSD === undefined) {
+            // no point in completing this if aUSD is not registered
+            this.skip();
+        }
+
+        const sudoThis = async (extrinsic: SubmittableExtrinsic<"promise", ISubmittableResult>) =>
```

Comment:
There already is a sudo utility [here](https://github.com/interlay/interbtc-api/blob/ad8c3035a6ae36865399993d7c2eeefdb7cff214/test/utils/helpers.ts#L75)

Reply:
Sadly not for `SubmittableExtrinsic<"promise", ISubmittableResult>` - only for `Promise<void>`.

So, in order to use that, I'd have to create a method returning `Promise<void>` which seems like more overhead than having this little helper.